WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -163,7 +168,8 @@
           <Border Classes="warning">
             <DockPanel>
               <PathIcon />
-              <TextBlock Text="Transaction uses unconfirmed coins." />
+              <TextBlock Text="Transaction uses unconfirmed coins." 
+                         ToolTip.Tip="This may cause your transaction to be delayed or rejected and you may not pay the desired amount of mining fees. Wait for a confirmation to only use confirmed coins." />
```

Comment:
```suggestion
                         ToolTip.Tip="This may cause your transaction to be rejected, or delay its confirmation, or cost more than needed to get it confirmed within the desired time. Wait for a confirmation to only use confirmed coins." />
```